### PR TITLE
added Randommeme command

### PR DESCRIPTION
**Pull request by Progame03**
Added Randommeme command alongside GSON as a Dependency using Maven
Had to be written without any real IDE so mistakes may have been made; code itself works

### CODE REVIEW

**Comment by Xirado** [comment]
Please change the line-ending in your IDE to CRLF to get proper diffs

File: src/main/java/at/xirado/bean/command/slashcommands/RandomMeme.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    public void executeCommand(@NotNull SlashCommandEvent event, @Nullable Member sender, @NotNull SlashCommandContext ctx)
+    {
+        URI uri = URI.create("https://meme-api.herokuapp.com/gimme/"+ Objects.requireNonNull(event.getOption("subreddit")).getAsString());
+        HttpRequest request = HttpRequest.newBuilder(uri).build();
+        String content = null;
+        try {
+            content = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString()).body();
+        } catch (IOException | InterruptedException e) {
+            e.printStackTrace();
+        }
```
Comment (Xirado) [inline src/main/java/at/xirado/bean/command/slashcommands/RandomMeme.java:41]: You should use `Bean.getInstance().getOkHttpClient()` to make REST requests

File: src/main/java/at/xirado/bean/command/slashcommands/RandomMeme.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        assert content != null;
```
Comment (Xirado) [inline src/main/java/at/xirado/bean/command/slashcommands/RandomMeme.java:42]: Don't use assert

File: src/main/java/at/xirado/bean/command/slashcommands/RandomMeme.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        if(content.contains("url")){
+            JsonObject jsonObject = new Gson().fromJson(content, JsonObject.class);
+            ctx.reply(jsonObject.get("url").getAsString()).queue();
```
Comment (Xirado) [inline src/main/java/at/xirado/bean/command/slashcommands/RandomMeme.java:45]: Use `DataObject.fromJson()` to map the json response

**Comment by Progame03** [comment]
> Please change the line-ending in your IDE to CRLF to get proper diffs

On it

**Comment by Xirado** [comment]
Github closed the PR since i force-pushed into master, you'd need to reopen another PR, sorry about that.

**Pull request closed without merging**